restapi.go

```diff
@@ -2328,3 +2350,141 @@ func (s *Session) RelationshipsMutualGet(userID string) (mf []*User, err error)
 	err = unmarshal(body, &mf)
 	return
 }
+
+// ------------------------------------------------------------------------------------------------
+// Functions specific to application (slash) commands
+// ------------------------------------------------------------------------------------------------
+
+// ApplicationCommandCreate creates an global application command. If guild is specified - creates guild specific application command.
+func (s *Session) ApplicationCommandCreate(appID string, cmd *ApplicationCommand, guildID string) (rcmd *ApplicationCommand, err error) {
+	if appID == "" {
+		appID = s.State.User.ID
+	}
+	endpoint := EndpointApplicationGlobalCommands(s.State.User.ID)
+	if guildID != "" {
+		endpoint = EndpointApplicationGuildCommands(s.State.User.ID, guildID)
+	}
+	body, err := s.RequestWithBucketID("POST", endpoint, *cmd, endpoint)
+	if err == nil {
```

Comment:
Please don't do this. Check all error. And best practices will be
```
body, reqErr := s.RequestWithBucketID("POST", endpoint, *cmd, endpoint)
if reqErr != nil {
    return reqErr
}
```

Reply:
Why? Who written that this is the standard?

Reply:
https://golang.org/doc/effective_go.html#if
![image](https://user-images.githubusercontent.com/18282288/104930057-8ec55780-599c-11eb-9793-17633a51ebd5.png)
The Go team wrote this standard. Please adhere to the Go standard when contributing to this repository. This repo adheres to the Go standard where possible, please adhere to the standards this repo follows

Reply:
Please adhere to the coding conventions that this repository uses, thank you.